src/sidebar/components/LoggedOutMessage.js

```diff
@@ -46,7 +46,7 @@ function LoggedOutMessage({ onLogin }) {
           aria-label="Hypothesis homepage"
           title="Hypothesis homepage"
         >
-          <Icon name="logo" classes="u-icon--xlarge hyp-u-color--grey-7" />
+          <Icon name="logo" classes="w-16 h-16 hyp-u-color--grey-7" />
```

Comment:
This is the only time in the app we render an icon really big (32px sq), so in this particular case I've opted for direct width and height dimensions. Again, gonna revisit icon sizing soon.